package.json

```diff
@@ -16,14 +16,40 @@
     "@types/bootstrap": "^3.3.0",
     "@types/d3": "^7.1.0",
     "@types/grecaptcha": "^3.0.1",
+    "@types/is-hotkey": "^0.1.1",
```

Comment:
I'm not sold on removing type checking for production because if it's possible to conditionally exclude modules based on environment, it's also possible to include modules and types and end up with some weirdy situation